R/vault.R

```diff
@@ -11,7 +11,7 @@ resolve_secrets <- function(x, config) {
       loadNamespace("vaultr")
       vault <- withr::with_envvar(
         orderly_envir_read(config$path),
-        vaultr::vault_client(addr = config$vault_server))
+        vaultr::vault_client(login = TRUE, addr = config$vault_server))
```

Comment:
This will look up `VAULTR_AUTH_METHOD` and throw an informative error message if that is not set - see tests